exe/launcher/Main.hs

```diff
@@ -62,9 +68,10 @@ Usage:
   cardano-wallet-launcher --version
 
 Options:
-  --network <STRING>           testnet, staging, or mainnet [default: testnet]
+  --network <STRING>           testnet, staging, mainnet, or local [default: testnet]
```

Comment:
why is `local` coming back?

Reply:
Because of Daedalus integrating with the wallet backend. They also need to be able to spin-up a local cluster for their own testing.